plugins/toc.md

```diff
@@ -10,11 +10,11 @@ controls: toolbar button, menu item
 {% assign pluginname = "Table of Contents" %}
 {% assign plugincode = "toc" %}
 
-`toc` plugin will generate basic *Table of Contents* and insert it into the editor at the current cursor position. Items for the table will be taken from the headers found in the content.
+The `toc` plugin will generate basic *Table of Contents* and insert it into the editor at the current cursor position. Items for the table will be taken from the headers found in the content.
```

Comment:
Other plugin names have not been `like this` they've just been named directly

Reply:
yeah, I can't fix all the docs in one day :) but yes.

Reply:
```suggestion
The Table of Contents plugin will generate basic *Table of Contents* and insert it into the editor at the current cursor position. Items for the table will be taken from the headers found in the content.
```